src/core/reactor_backend.cc

```diff
@@ -1146,7 +1146,13 @@ bool reactor_backend_selector::has_enough_aio_nr() {
      * So this method calculates:
      *  Available AIO on the system - (request AIO per-cpu * ncpus)
      */
-    if (aio_max_nr - aio_nr < reactor::max_aio * smp::count) {
+    unsigned _smp_local_count = 1;
```

Comment:
We don't prepend local variables names with `_`

Reply:
Could you point me to the relevant documentation where this is specified? I took a quick glance at the code and there are local variable names starting with a `_`. No problem over changing the variable name as long as it is indeed a practice. :)

Reply:
It's indeed a practice. `_` usually comes as a prefix for private class members, not for class variables unless there's a _really_ _good_ reason for it.